src/ChannelAllocationChromosome.java

```diff
@@ -4,9 +4,22 @@
 class ChannelAllocationChromosome implements IChromosome {
 
     public List<Float> genes;
```

Comment:
I changed the visibility of this member to private in my recent commit, please handle conflicts.